ProjectBank/Server.Tests/Controllers/ProjectsControllerTests.cs

```diff
@@ -0,0 +1,133 @@
+namespace Server.Tests;
+
+public class ProjectsControllerTests
+{
+
+    [Fact]
+    public async Task Post_creates_Project()
+    {
+        // Arrange
+        var toCreate = new CreateProjectDTO();
+        var created = new ProjectDTO(1, "Very nice project", "Active", "1", "A created project", DateTime.UtcNow, DateTime.UtcNow, new List<string>(), new List<UserDTO>());
+        var repository = new Mock<IProjectRepository>();
+        repository.Setup(m => m.CreateProjectAsync(toCreate)).ReturnsAsync(created);

```

Comment:
I dont understand this line :/

Reply:
It means that we mock what the CreateProjectAsync does, we just tell it that when this method is run it should return the created, which we have previously defined. That means that it doesn't actually execute the code/method CreateProjectAsync.